client/components/select-vertical/suggestion-search.tsx

```diff
@@ -131,7 +131,7 @@ const SelectVerticalSuggestionSearch: FC< Props > = ( {
 
 		return suggestions.concat( [
 			{
-				value: '',
+				value: '-1',
```

Comment:
Could you add the comment to explain why we have to give it `-1` and link to the issue?

Reply:
Changing the value to -1 here will cause the API to save the site options `site_vertical` as -1 as well. As result, next time the user access this page the front-end will try to fetch `https://public-api.wordpress.com/wpcom/v2/site-verticals/-1` which is an invalid route. 

To solve this, I think we can either only pass -1 to Tracks for analytics purposes, or pass 0 and map that to "Something else". I prefer the former approach, since the vertical API doesn't have the concept of "Something else". It's purely a front-end construct.

Reply:
> To solve this, I think we can either only pass -1 to Tracks for analytics purposes
That was my origin intention, but it's seems I put it on wrong place as you explained, will update and keep you posted